src/js/main.js

```diff
@@ -0,0 +1,8 @@
+import {startGame} from "./handleGame.js";
+
+const gameBigBoard = document.getElementById("bigBoard");
```

Comment:
1. the expected "GameView" should find the elements itself, and main.js should just start a new game, something like the following 3 liners:

- const game = new Game();
- const gameView = new GameView(game);
- gameView.start();

2. You sometimes use jquery, sometimes document.getElementById - please be consistent